translations/hu/get5.phrases.txt

```diff
@@ -340,8 +388,116 @@
     {
         "hu"            "[NEM ÁLL KÉSZEN]"
     }
+    "MapSelectionTeamChatOnly"
+    {
+        "hu"            "Csak csapat csevegőben írhatsz, pályaválasztás alatt."
+    }
+    "MapSelectionInvalidMap"
+    {
+        "hu"            "{1} nem érvényes pálya."
+    }
+    "MapSelectionPickMapHelp"
+    {
+        "hu"            "Használd a {1} <pálya> parancsot, hogy pályát kiválaszd."
+    }
+    "MapSelectionBanMapHelp"
+    {
+        "hu"            "Használd a {1} <pálya> parancsot, hogy pályát eltávolítsd."
+    }
+    "MapSelectionPickSideHelp"
+    {
+        "hu"            "Használd a {1} vagy a {2} parancsot az oldalválasztáshoz."
+    }
+    "MapSelectionAnnounceCaptains"
+    {
+        "hu"            "Csapatkapitánya a {1}: {2}"
+    }
+    "MapSelectionBan"
+    {
+        "hu"            "ELTÁVOLÍT"
+    }
+    "MapSelectionPick"
+    {
+        "hu"            "VÁLASZT"
+    }
+    "MapSelectionTurnToPick"
+    {
+        "hu"            "{1} most {2} egy pályát. {3} pályának"
```

Comment:
Is this right? It's supposed to say "TeamName picked de_dust as map number 2." for instance. It seems this full-stop is in a weird place? @enerbewow @Nymu5  ?

Reply:
No. Sorry I didn't saw it. after `pályát` don't need that dot. 
team_enerbewow most választott pályát 1. pályának - team_enerbewow picked a map for `number of map` map
I actually don't know the text behind {3} but i'll check it.

Reply:
{1} must now {2} a map to play as map {3}.
`team_name` must now `picked` a map to play as map `map_number`

Reply:
so in mine/our translation.
`team_name` most választott egy pályát `map_number` pályának
in raw translate `team_name` now `picked` a map for `map_number` pályának.